apps/fluent-tester/src/RNTester/TestComponents/FocusZone/FocusZoneTest.tsx

```diff
@@ -1,80 +1,36 @@
 import * as React from 'react';
 import { View, Picker, Switch } from 'react-native';
-import { FocusZone, Button, Text, RadioGroup, RadioButton, FocusZoneDirection, NavigateAtEnd, Checkbox } from '@fluentui/react-native';
+import { FocusZone, Button, Text, RadioGroup, RadioButton, FocusZoneDirection, Checkbox } from '@fluentui/react-native';
 import { Test, TestSection, PlatformStatus } from '../Test';
 import { FOCUSZONE_TESTPAGE } from './consts';
 import { focusZoneTestStyles, stackStyleFocusZone } from './styles';
 import { commonTestStyles } from '../Common/styles'
 import { Stack } from '@fluentui-react-native/stack';
 
-const navigation2DFocusZone: React.FunctionComponent<{}> = () => {
-  const buttonRef = React.useRef<View>(null);
-
-  // const onFocus = () => {
-  //   console.group('hu');
-  // }
-
-  return (
-    <View>
-      <View>
-        <Button content="Outside FocusZone" />
-      </View>
-      <FocusZone use2DNavigation={true} defaultTabbableElement={buttonRef} navigateAtEnd='NavigateWrap'>
-        <View style={focusZoneTestStyles.focusZoneContainer}>
-          <View style={focusZoneTestStyles.focusZoneViewStyle}>
-            <Button content="#1" style={focusZoneTestStyles.focusZoneButton} />
-            <Button content="#5" style={focusZoneTestStyles.focusZoneButton} />
-            <Button content="#5" style={focusZoneTestStyles.focusZoneButton} />
-          </View>
-          <View style={focusZoneTestStyles.focusZoneViewStyle}>
-            <Button style={focusZoneTestStyles.focusZoneButton} content="#2" />
-            <Button componentRef={buttonRef} style={focusZoneTestStyles.focusZoneButton} content="Test" />
-            <Button content="#5" style={focusZoneTestStyles.focusZoneButton} />
-          </View>
-          <View style={focusZoneTestStyles.focusZoneViewStyle}>
-            <Button style={focusZoneTestStyles.focusZoneButton} content="#3" />
-            <Button style={focusZoneTestStyles.focusZoneButton} content="Test" />
-            <Button content="#5" style={focusZoneTestStyles.focusZoneButton} />
-          </View>
-        </View>
-      </FocusZone>
-      <View>
-        <Button content="Outside FocusZone" />
-      </View>
-    </View >
-  );
-};
-
-const commonUsageFocusZone: React.FunctionComponent<{}> = () => {
+const directionalFocusZone: React.FunctionComponent<{}> = () => {
   return (
     <View>
       <Button content="Outside FocusZone" />
-      <FocusZone disabled={true}>
-        <Checkbox label="disabled" />
-        <Checkbox label="disabled" />
-        <Checkbox label="disabled" />
-      </FocusZone>
-      <Button content="Outside FocusZone" />
-      <RadioGroup label="FocusZone RadioGroup with Circular Navigation" defaultSelectedKey="A">
-        <FocusZone navigateAtEnd='NavigateWrap'>
+      <FocusZone disabled={null} focusZoneDirection='bidirectional'>
```

Comment:
Why are you setting disabled={null} ?

Reply:
This was just for testing purposes to ensure the ViewManager was working correctly. I'll remove it

Reply:
Removed in newest commit